views/js/controller/items/action.js

```diff
@@ -63,11 +63,13 @@ define([
             uri: actionContext.id
         };
         const config = _.merge(defaultConfig, module.config());
-
-        previewerFactory(config.provider, config.uri, config.state, {
+        const previewerConfig = _.omit({
             readOnly: false,
-            fullPage: true
-        });
+            fullPage: true,
+            pluginsOptions: config.pluginsOptions
```

Comment:
FYI: this is wrong, didn't notice that this module is applied for delivery too, not just for previewer. Module config  must be moved to another place.